ComposeStarter/app/src/test/java/presentation/GreetingScreenTest.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package presentation
+
+import com.example.android.wearable.composestarter.presentation.GreetingScreen
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.ParameterizedRobolectricTestRunner
+
+@RunWith(ParameterizedRobolectricTestRunner::class)
+class GreetingScreenTest(override val device: WearDevice) : WearScreenshotTest() {
+
+    @Test
+    fun greetingScreenTest() = runTest {
+        GreetingScreen(greetingName = "screenshot", onShowList = {})
```

Comment:
Can we wrap with 

```
AppScaffold {}
```

So we get TimeText and PositionIndicator?

Reply:
Done!